gdk4/Gir.toml

```
@@ -476,6 +480,10 @@ manual_traits = ["SurfaceExtManual"]
     manual = true
     doc_trait_name = "SurfaceExtManual"
     [[object.function]]
+    name = "set_input_region"
+    manual = true #mutability issue
+    doc_trait_name = "SurfaceExtManual"
```

```suggestion
    [[object.function.parameter]]
    name = "region"
    const = true
```
what about something like this & dropping the manual code?